2.Objects and Functions/script.js

```diff
@@ -0,0 +1,68 @@
+// object task
+
+/*let car = {
+    color: 'black'
+}
+
+car.color = 'green';
+car.engine = function func() { console.log("engine power") };
+*/
+
+//Pears and apples
+
+/*let pears = 12;
+let apples = 22;
+function sum(x, y) {
+    return x + y;
+}
+sum(pears, apples);
+*/
+
+//payment terminal
+
+/*
+let name = prompt("Enter Name: ");
+function nameDefine(x) {
```

Comment:
In this task, you need to make the name match the requirement, and not just enter the name.